template/gem_name.gemspec.erb

```diff
@@ -30,16 +30,16 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['lib/**/*', 'README.md', 'LICENSE.md', 'CHANGELOG.md']
 
-  spec.required_ruby_version = '>= 2.5', '< 4'
+  spec.required_ruby_version = '>= 2.6', '< 4'
```

Comment:
I also don't look at this as related to this PR, especially in the single commit. Why?

Reply:
Fair enough, this was outside of the original scope and I failed to make it clear in my commit.
What started originally as a simple addition to the CI, became a "let's modernize the CI action a bit so new gems will be more up-to-date"